### ISSUE TITLE
Imported and not used / undefined imports in errors

### ISSUE THREAD

**Issue by jakebailey** [goccmack/gocc#87]
After regenerating:

```
internal/cbparse/errors/errors.go:6:2: imported and not used: "bytes"
internal/cbparse/errors/errors.go:24:11: undefined: strings
internal/cbparse/errors/errors.go:45:11: undefined: strings
```

**Comment by jakebailey** [comment]
Similarly in `parser`.

**Comment by jakebailey** [comment]
Ah, forgive me for not looking; this is fixed if #86 is merged.
